### PR TITLE
Structure changes

### PR DESCRIPTION
**Pull request by one-some**
This PR:
- Obliterates `utils.HAS_ACCELERATE` to make maintaining code easier. Accelerate has been a requirement for a while.
- Splits up `load_model` and other `aiserver.py` inference code into a million bajillion pieces: 
    - Models are now represented by a `InferenceModel`-based class with API-user-facing `load()`, `core_generate()` and `raw_generate()` functions. Each model should be located in its own `modeling/inference_models/SUPER_DUPER_MODEL.py` file.
    - The previously transformers-only StoppingCriteria stoppers and token hooks have been replaced with generic `Stoppers` and `PostTokenHooks` methods. These can be implemented on any model type with relative ease and a reference implementation can be found in `modeling/inference_models/hf_torch.py`.
    - Samplers are now much more universal and are no longer tied to Transformers (as far as I can see). All sampler code is now located in `modeling/warpers.py` and each sampler is structured into a singleton class with `torch`, `jax_dynamic`, `jax_static`, and `value_is_valid` methods. This should make creating samplers easier, but more importantly, make implementing them into unsupported models easier.
- Zaps the old, hacky, and crusty RWKV implementation into oblivion. Nobody (to my knowledge) was using it due to the sheer impracticality and its concealment behind the Experimental toggle. A new implementation is present and more functional than the last one, but still not user-ready, and thus hidden behind the Experimental toggle.
- [Moves `aetherroom.club` import code out of aiserver.py](https://github.com/one-some/KoboldAI-united/pull/1) (by @whjms)
- Adds unit tests for model loading and inference (`modeling/test_generation.py`)
- Makes modellist (in `aiserver.py`) oriented around objects rather than lists. This makes working with and expanding the model menu a lot easier
- [Adds API endpoint for custom stop sequences](https://github.com/one-some/KoboldAI-united/pull/5) (by @LostRuins)

This will need a lot of testing. I have been testing extensively (and will continue to), but the bugs have started to look normal to me at this point :^)

### CODE REVIEW

**Comment by one-some** [comment]
This should be ready for review

**Comment by one-some** [comment]
...or not. found a particularly nasty bug with sampling or determinism or something. whatever the case its busted. probably skipping over some hf score manipulation thing by overriding it with our samplers

**Comment by one-some** [comment]
ok now its ready for real (no fake)

File: modeling/inference_model.py [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+try:
+    import tpu_mtj_backend
+except ModuleNotFoundError as e:
+    # Not on TPU... hopefully
+    if utils.koboldai_vars.use_colab_tpu:
+        raise e
+
+# I don't really like this way of pointing to the current model but I can't
+# find a way around it in some areas.
+current_model = None
```
Comment (whjms) [inline modeling/inference_model.py]: would having `HFTorchInferenceModel` just access the tokenizer with `self.tokenizer` work?

Comment (one-some) [reply modeling/inference_model.py]: looks like it! i think this was a byproduct of originally not having the patch functions be methods of `HFTorchInferenceModel`, but now they are. good catch!

**Pull request merged**